docker-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -eu
+set -o pipefail
+
+trap finish EXIT INT TERM
```

Comment:
This could also be moved into the Node script, but I felt the `trap` would be more suitable